docs/UserGuide.md

```diff
@@ -86,174 +92,217 @@ Shows a message explaining how to access the help page.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A volunteer can have any number of tags 
+A volunteer can have any number of tags, medical qualifications and available dates. 
 </div>
 
 Examples:
 
-* `add_volunteer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 ag/23 vnr/S8457677H re/WEST`
-* `add_volunteer n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 ag/27 vnr/S8959886I t/experienced`
-* `add_volunteer n/John Wick e/johnwick@example.com a/New yourk p/1234561 vnr/T1254567D ag/28 dr/2023-04-01,2023-04-15`
-* `add_volunteer n/Sally Tan re/CENTRAL`
+* `add_volunteer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 ag/23 vnr/S8457677H re/WEST t/graduate t/new`
+* `add_volunteer n/Betsy Crowe p/1234567 ag/27 vnr/S8959886I re/NORTH t/experienced mt/CPR ADVANCED mt/BLS BASIC dr/2023-06-03,2023-06-17`
+* `add_volunteer n/John Wick e/johnwick@example.com a/New yourk p/1234561 vnr/T1254567D ag/28 dr/2023-04-01,2023-04-15 dr/2023-03-01,2023-07-17`
+* `add_volunteer n/Sally White ag/23 vnr/S8457677H`
 
-### Editing a volunteer by index : `edit_volunteer`
 
-Edits an existing volunteer based on their index in the volunteers list.
+#### Pair volunteer and elderly: `add_pair`
 
-Format: `edit_volunteer INDEX [n/NAME] [nr/NRIC] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [ag/AGE] [re/REGION] [mt/MEDICAL_TAG] [t/TAG]… [dr/START_DATE,END_DATE]…`
+Add a pairing between an existing elderly and volunteer.
+This allows you to track which elderly members are assigned to which volunteers.
 
-* Edits the volunteer at the specified `INDEX`. The index refers to the index number shown in the displayed volunteers list. The index **must be a positive integer** 1, 2, 3, …​
-* Any combination of the optional fields is possible but **at least one** optional field must be specified.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the volunteer will be removed i.e adding of tags is not cumulative.
-* You can remove all the volunteer’s tags by typing `t/` without
-  specifying any tags after it.
-* Same behavior as tags for available dates.
+Format: `add_pair enr/ELDERLY_NRIC vnr/VOLUNTEER_NRIC`
+
+* After pairing, the newly added pairs appear in the pair list in the window.
+* Only elderly members and volunteers existing in FriendlyLink's data can be paired.
+* Only elderly members and volunteers with intersecting available dates can be paired.
+* Elderly member and volunteers in different regions can be paired but a warning message is issued.
+* Duplicate pairs will fail to be added to FriendlyLink.
+* Alphabets in NRIC are case-insensitive.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes pairing**<br>
+
+* Pairing will only be successful if there are no clashes in availability that is specified by the elderly and volunteer.
```

Comment:
I believe this restriction is being removed. See #140

Reply:
#140 is not "allowing pairing with no overlapping available dates", but "after edit, one pair may have no overlapping available dates, warn such edits"

Reply:
I feel that add pair should completely block if available dates don't match but also throw a warning like edit. Need @gohyongjing to confirm.

Reply:
yeap, my pr #150 makes it such that `editElderly`, `editVolunteer` and `addPair` all accepts pair with no matching dates and regions but gives a notice in the success message

Reply:
Ok I will then update it